RealmSwift/Migration.swift

```diff
@@ -0,0 +1,220 @@
+    RLMRealm.setDefaultRealmSchemaVersion(schemaVersion, withMigrationBlock: accessorMigrationBlock(migrationBlock))
+}
+
+/**
+Specify a schema version and an associated migration block which is applied when
+opening a Realm at the specified path with an old schema version.
+
+Before you can open an existing `Realm` which has a different on-disk schema
+from the schema defined in your object interfaces, you must provide a migration
+block which converts from the disk schema to your current object schema. At the
+minimum your migration block must initialize any properties which were added to
+existing objects without defaults and ensure uniqueness if a primary key
+property is added to an existing object.
+
+You should call this method before accessing any `Realm` instances which
+require migration. After registering your migration block, Realm will call your
+block automatically as needed.
+
+:param: version   The current schema version.
+:param: realmPath The path of the Realms to migrate.
+:param: block     The block which migrates the Realm to the current version.
+*/
+public func setSchemaVersion(schemaVersion: UInt, realmPath: String, migrationBlock: MigrationBlock) {
+    RLMRealm.setSchemaVersion(schemaVersion, forRealmAtPath: realmPath, withMigrationBlock: accessorMigrationBlock(migrationBlock))
+}
+
+/**
+Get the schema version for a Realm at a given path.
+:param: realmPath     Path to a Realm file.
+:param: encryptionKey Optional 64-byte encryption key for encrypted Realms.
+:param: error         If an error occurs, upon return contains an `NSError` object
+                      that describes the problem. If you are not interested in
+                      possible errors, omit the argument, or pass in `nil`.
+:returns: The version of the Realm at `realmPath` or `nil` if the version cannot be read.
+*/
+public func schemaVersionAtPath(realmPath: String, encryptionKey: NSData? = nil, error: NSErrorPointer = nil) -> UInt? {
+    let version = RLMRealm.schemaVersionAtPath(realmPath, encryptionKey: encryptionKey, error: error)
+    if version == RLMNotVersioned {
+        return nil
+    }
+    return version
+}
+
+/**
+Performs the registered migration block on a Realm at the given path.
+
+This method is called automatically when opening a Realm for the first time and does
+not need to be called explicitly. You can choose to call this method to control
+exactly when and how migrations are performed.
+
+:param: path          The path of the Realm to migrate.
+:param: encryptionKey Optional 64-byte encryption key for encrypted Realms.
+                      If the Realms at the given path are not encrypted, omit the argument or pass
+                      in `nil`.
+
+:returns: `nil` if the migration was successful, or an `NSError` object that describes the problem
+          that occured otherwise.
+*/
+public func migrateRealm(path: String, encryptionKey: NSData? = nil) -> NSError? {
```

Comment:
how about using the default realm path as a default parameter value? so users using the default realm can just call `migrationRealm()`